bpf/bpf_lxc.c

```diff
@@ -1033,7 +1043,8 @@ __section_tail(CILIUM_MAP_POLICY, LXC_ID) int handle_policy(struct __sk_buff *sk
 
 	ifindex = skb->cb[CB_IFINDEX];
 
-	send_trace_notify(skb, TRACE_TO_LXC, src_label, SECLABEL, LXC_ID, ifindex);
+	send_trace_notify(skb, TRACE_TO_LXC, src_label, SECLABEL, LXC_ID, ifindex,
+			  TRACE_REASON_POLICY);
```

Comment:
This would mean that for any ingress flow record, the reason is always set to policy. Given the use at egress, this would not be correct as it can be allowed because of established/reply/related as well. We can pass in a `int *forwarding_reason` into `ipv4_policy()` and `ipv6_policy()` to retrieve the conntrack state as for egress.

Reply:
Done.